.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+## Summary
+A short description of the changes with a link to an open issue.
+
+## Checklist
+Delete items not relevant to your PR:
+- [ ] Unit and integration tests covering the common scenarios were added
+- [ ] A human-readable description of the changes was provided to include in CHANGELOG
```

Comment:
Should it fill a contributor? It leads to conflicts between PRs and wrong placement (if PR is open while a new version is released) =(